src/reconcile/rec_col.c

```diff
@@ -632,23 +637,12 @@ __wt_rec_col_var(
     cbt = &r->update_modify_cbt;
     cbt->iface.session = (WT_SESSION *)session;
 
-    /*
-     * Acquire the newest-durable timestamp for this page so we can roll it forward. If it exists,
-     * it's in the WT_REF structure or the parent's disk image.
-     */
-    if ((addr = pageref->addr) == NULL)
-        newest_durable_ts = WT_TS_NONE;
-    else if (__wt_off_page(pageref->home, addr))
-        newest_durable_ts = addr->stop_durable_ts;
-    else {
-        __wt_cell_unpack(session, pageref->home, pageref->addr, vpack);
-        newest_durable_ts = vpack->newest_stop_durable_ts;
-    }
-
     /* Set the "last" values to cause failure if they're not set. */
```

Comment:
I don't understand why this code is no longer needed.

Similar code was removed in row-store reconciliation.

Reply:
Earlier we didn't store the durable timestamp as part of the `value` cell and only it is saved in the `address` cell. Whenever the same page is getting reconciled again, we take the `newest durable timestamp` stored in the address cell as the default value for each `value` cell on the page. But now as we are storing the durable timestamp as part of the `value` cell, we don't need this special code.